.github/workflows/pr-bundle-diff-checks.yaml

```diff
@@ -102,5 +103,5 @@ jobs:
         run: |
           echo "Checking that the RBAC roles of the downstream RHDH operator are not out of sync with the upstream CSV..."
           diff -U 1 \
-            <(yq '.spec.install.spec.clusterPermissions' bundle/manifests/backstage-operator.clusterserviceversion.yaml | grep -v 'serviceAccountName: ') \
+            <(yq '.spec.install.spec.clusterPermissions' bundle/rhdh/manifests/backstage-operator.csv.yaml | grep -v 'serviceAccountName: ') \
```

Comment:
did this path actually change? seems like this is only in 1.4 

https://github.com/redhat-developer/rhdh-operator/tree/release-1.3/bundle/rhdh/manifests is 404'd